servicetalk-loadbalancer-experimental/src/main/java/io/servicetalk/loadbalancer/OutlierDetectorConfig.java

```diff
@@ -558,6 +557,19 @@ public Builder baseEjectionTime(final Duration baseEjectionTime) {
             return this;
         }
 
+        /**
+         * Set the ejection time jitter.
+         * Defaults to 3 seconds.
+         * @param ejectionTimeJitter the maximum jitter to add to the calculated ejection time.
+         * @return {@code this}.
+         */
+        public Builder ejectionTimeJitter(final Duration ejectionTimeJitter) {
```

Comment:
If you'd like to associate it with `baseEjectionTime`, consider using `@see` javadoc tag to draw a relationship